test/common.bash

```diff
@@ -79,6 +79,15 @@ prerequisites_ok () {
     fi
 }
 
+need_squashfs () {
+    ( command -v mksquashfs >/dev/null 2>&1 ) || skip "no squashfs-tools found"
+    ( command -v squashfuse >/dev/null 2>&1 ) || skip "no squashfuse found"
```

Comment:
Currently the Centos package of squashfuse contains both `squashfuse` and `squashfuse_ll` but the debian package only has `squashfuse` the tests and scripts should still work even if they only have squashfuse.

Reply:
Is there a situation where one would have `squashfuse_ll` but not `squashfuse`?

Reply:
Not that I am aware of. Unless you build from source and manually delete `squashfuse` leaving only `squashfuse_ll`

Reply:
As a note, when building from source it will only build the high level `squashfuse` if certain requirements aren't met for the low level `squashfuse_ll` so even if building from source you would have to really go out of your way to not have `squashfuse` but still have `squashfuse_ll`